pkgs/by-name/ts/tslib/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  clangStdenv,
+  fetchFromGitHub,
+  cmake,
+}:
+
+clangStdenv.mkDerivation (finalAttrs: {
```

Comment:
is there a specific reason why the `clangStdenv` is used instead of `stdenv`?

Reply:
https://github.com/NixOS/nixpkgs/pull/341643#pullrequestreview-2315315476

Reply:
while `alive2` specifically depends on clang, `tslib` does not (and contains a GCC feature test in the configure script)

Reply:
Ah I thought it depended on clang.